unblob/handlers/archive/hp/ipkg.py

```diff
@@ -0,0 +1,111 @@
+import io
+from pathlib import Path
+from typing import Iterable, Optional, Tuple
+
+from dissect.cstruct import Instance
+from structlog import get_logger
+
+from unblob.extractor import carve_chunk_to_file
+from unblob.file_utils import Endian, File, InvalidInputFormat, StructParser, snull
+from unblob.models import Chunk, Extractor, HexString, StructHandler, ValidChunk
+
+logger = get_logger()
+
+
+C_DEFINITIONS = r"""
+    typedef struct ipkg_file_entry {
+        char file_name[256];
+        uint64 offset;
+        uint64 size;
+        char unknown[4];
```

Comment:
maybe/probably a CRC32 as a `uint32` ?

Reply:
as discussed, sometimes the CRC is valid, sometimes it isn't, we decided not to validate it